CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- Updated team ownership to be provider dependant
```

Comment:
```suggestion
- Update team ownership for KaaS-related alerts to be provider dependent.
```
We try to use present tense and punctuation on change descriptions in general (also fixed a typo).